cheatsheets/gleam-for-erlang-users.md

```diff
@@ -459,7 +459,7 @@ All strings in Gleam are UTF-8 encoded binaries.
 ### Tuples
 
 Tuples are very useful in Gleam as they're the only collection data type that allows for mixed
-types of elements in the collection. The syntax for a tuple literal - `tuple("a", "b")` - can be
+types of elements in the collection. The syntax for a tuple literal - `#("a", "b")` - can be
```

Comment:
The new tuple syntax most probably can't be confused for a function call 😛

Reply:
Well spotted, thanks!